docs/public-networks/concepts/transactions/pool.md

```diff
@@ -53,6 +53,26 @@ If you previously configured transaction pool behavior, upgrade to the layered t
 You can opt out of the layered transaction pool implementation by setting the
 [`--tx-pool`](../../reference/cli/options.md#tx-pool) option to `sequenced`.
 
+### Penalize invalid pending transactions
```

Comment:
for clarity, we should surface in the name that these are _transient_ invalid pending transactions, since we already has the concept of _final_ invalid pending transactions, that are removed from the pool immediately since their status cannot change.

Reply:
Thanks. Done.